cogni-sem/src/test/java/zone/cogni/sem/jena/SparqlUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package zone.cogni.sem.jena;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import zone.cogni.libs.sparqlservice.impl.JenaModelSparqlService;
+
+public class SparqlUtilsTest {
+  private final JenaModelSparqlService jenaModelSparqlService = new JenaModelSparqlService();
+
+  @Test
+  public void test() {
+    initializeSource();
+    confirmResult();
+  }
+
+  private void initializeSource() {
+    jenaModelSparqlService.executeUpdateQuery(
+      "PREFIX jolux: <http://data.legilux.public.lu/resource/ontology/jolux#> " +
+      "INSERT DATA { " +
+      "  GRAPH <https://fedlex.data.admin.ch/eli/oc/2023/423/legal-analysis/graph> { " +
+      "    <https://fedlex.data.admin.ch/eli/oc/2023/423/legal-analysis> a jolux:LegalAnalysis. " +
+      "    <https://fedlex.data.admin.ch/eli/oc/2023/423/legal-analysis> jolux:impactFromLegalResourceComment " + "'" + SparqlUtils.escapeLiteral("A\ncomment") + "'" +
+      "  }" +
```

Comment:
check the doc natan sent, and do more tests based on that
also, just any uri with just 1 literal is needed
and i would than have a method that receives the string, clears the dataset, adds it, re-fetches it and asserts

your test method can run a bunch of tests for the different to-escape characters